integration-tests/initiateDataRequest.spec.ts

```diff
@@ -4,72 +4,112 @@ import { approveZendeskTicket } from './utils/zendesk/approveZendeskTicket'
 import { deleteZendeskTicket } from './utils/zendesk/deleteZendeskTicket'
 import { invalidRequestData, validRequestData } from './constants/requestData'
 import {
+  ANALYSIS_BUCKET_NAME,
+  AUDIT_BUCKET_NAME,
   INITIATE_DATA_REQUEST_LAMBDA_LOG_GROUP,
-  PROCESS_DATA_REQUEST_LAMBDA_LOG_GROUP
+  INTEGRATION_TEST_DATE_PREFIX,
+  PROCESS_DATA_REQUEST_LAMBDA_LOG_GROUP,
+  TEST_FILE_NAME
 } from './constants/awsParameters'
 import { FilteredLogEvent } from '@aws-sdk/client-cloudwatch-logs'
 import { getZendeskTicket } from './utils/zendesk/getZendeskTicket'
 import { listZendeskTicketComments } from './utils/zendesk/listZendeskTicketComments'
+import { deleteAuditData } from './utils/aws/s3DeleteAuditData'
+import { copyAuditDataFromTestDataBucket } from './utils/aws/s3CopyAuditDataFromTestDataBucket'
 
 describe('Submit a PII request with approved ticket data', () => {
```

Comment:
It looks like we are now testing `initiateDataRequest` and `processDataRequest` functionality here. Have we decided to now test longer flows rather than "slices"?

Reply:
That's not really been discussed yet. But if we wanted to break them up into slices, that can easily be done in a separate PR.